### PR TITLE
Add quicker sign functionality to the e-Manifest Python package

### LINKED ISSUE #1535: Remove extra slash in RcrainfoClient base_url
The RCRAInfoClient appends an additional forward slash in the request URL, after `rest` and before `api`

https://rcrainfopreprod.epa.gov/rcrainfo/rest//api/v1/emanifest/manifest/100033134ELC

The request is made fine by the emanifest library, but I'm mocking API requests and the extra slash is causing problems. We should go ahead and fix the RcrainfoClient URL so it's accurate rather than rely on the RCRAInfo server to resolve our error.

### PR DESCRIPTION
**Pull request by wiljnichepa**
This PR adds the Quicker Sign functionality to the Python package. It also adds an example JSON file to the testing documentation.

This PR also addresses the duplicative "/" in the RcrainfoClient base_url to close #1535

### CODE REVIEW

File: emanifest-py/src/emanifest/client.py [hunk 1]
```diff
@@ -841,6 +841,24 @@ def delete_manifest(self, mtn: str) -> RcrainfoResponse:
         """
         endpoint = f'{self.base_url}/api/v1/emanifest/manifest/delete/{mtn}'
         return self.__rcra_request('DELETE', endpoint)
+    
+    def quicker_sign_manifest(self, reg: bool = False, **kwargs) -> RcrainfoResponse:
```
Comment (dpgraham4401) [inline emanifest-py/src/emanifest/client.py:845]: We don't need to `reg` argument, there's not a regulator version of Quicker Sign.

Should we shorten the function signature to `def sign_manifest(...)`? it's a little less explicit but shorter and I can't imagine there will be a **Quicker-er** sign in the foreseable future.

File: emanifest-py/src/emanifest/client.py [hunk 2]
```diff
@@ -841,6 +841,24 @@ def delete_manifest(self, mtn: str) -> RcrainfoResponse:
...
+            siteType (str) : The site on the manifest that is signing (Generator, Tsdf, Transporter, RejectionInfo_AlternateTsdf). Case-sensitive
+            printedSignatureName (str) : The name of the person who signed the manifest
+            printedSignatureDate (date) : The date the person signed the manifest
+            transporterOrder (int) : If the site is a transporter, the order of that transporter on the manifest
+            
+        Returns:
+            dict: message of success or failure
+        """
+        endpoint = f'{self.base_url}/api/v1/emanifest/manifest/quicker-sign'
+        return self.__rcra_request('POST', endpoint)
```
Comment (dpgraham4401) [inline emanifest-py/src/emanifest/client.py:861]: we need to pass the keyword arguments like 
```python
return self.__rcra_request('POST', endpoint, **kwargs)
 ```

**Review by dpgraham4401 — CHANGES_REQUESTED** [review]

File: emanifest-py/src/emanifest/client.py (hunk 1 above)
Comment (wiljnichepa) [reply emanifest-py/src/emanifest/client.py:845]: Good point. Let's shorten it. Additional signing functionality would likely be a modification of this anyway.

**Review by dpgraham4401 — APPROVED** [review]
Awesome Possum!

**Comment by dpgraham4401** [comment]
Plz merge when ready, cheers!

**Pull request merged**